### DISCUSSION TITLE
Is it possible to disable the pre-rendering in a specific page?

### DISCUSSION THREAD

**Discussion by 64J0** [vercel/next.js#16297] [Help] [answered]
Hi guys, hope you're well. 
I'm trying to make a page in my blog and it needs to **NOT** be pre-rendered in any way. The concept of pre-reder is presented in the tutorial (https://nextjs.org/learn/basics/data-fetching/pre-rendering). 
I need that this page assumes a normal React behavior, so the page will not be pre-rendered and the browser of the user will Hydrate the code with local javascript. Is it possible?
Thanks guys.

**Comment by jamesmosier** [comment, marked as answer]
You can opt-out of pre-rendering by using `getInitialProps`. See this issue for more info https://github.com/vercel/next.js/issues/16124

**Reply by 64J0** [reply]
Alright, I'll try it. Thanks.

**Reply by mkappelhof** [reply]
Thanks for the quick feedback, will try that
